### DISCUSSION TITLE
Sqinn-Go is a Golang library for accessing SQLite databases in pure Go

### DISCUSSION THREAD

**Discussion by dschissler** [pocketbase/pocketbase#3448] [Show and tell]
@ganigeorgiev 

* https://github.com/cvilsmeier/sqinn-go
* https://news.ycombinator.com/item?id=37769633

**Comment by ganigeorgiev** [comment]
Hm, interesting approach with the forked process.

We can't use it though because we rely on the standard `database/sql` drivers abstraction (in the future there are even plans to allow devs to define their own [`connectDB`](https://github.com/pocketbase/pocketbase/blob/837134559fe048e24fb1eb129c5c10d218f7ef3d/core/db_cgo.go#L43) handler to load extensions or a custom SQLite driver like [sqlcipher](https://github.com/mutecomm/go-sqlcipher)).
